assembl/static2/js/app/components/search.jsx

```diff
@@ -270,12 +273,14 @@ const PostHit = ({ bemBlocks, collapseSearch, locale, result }) => {
   const body = highlightedLSOrTruncatedLS(result, 'body', locale);
   const published = {};
   const ideaTitle = highlightedLSOrTruncatedLS(result, 'idea_title', locale);
+  const postUrl = getUrl(result);
+  const ideaUrl = postUrl.replace(postUrl.slice(postUrl.indexOf('#'), postUrl.length), '');
```

Comment:
`const ideaUrl = postUrl.slice(0, postUrl.indexOf('#'));`
is simpler no?

Reply:
💯